File: Reverse String/ans1.java
public class Solution {
    public String reverseString(String s) {
		char[] b = s.toCharArray();
		char[] reverseArray = new char[b.length];
		
		for (int i = 0; i < b.length; i++){
			reverseArray[b.length - 1 - i] = b[i];
		}
		String ans = new String(reverseArray);
		return ans;
    }
}

File: Number Complement/ans1.py
class Solution(object):
    def findComplement(self, num):
        """
        :type num: int
        :rtype: int
        """
        
        binary_list = list(str(bin(num)))

        ans = []
        
        for i in range(2, len(binary_list), 1):
        	if binary_list[i] != str(0):
        		ans.append('0')
        	else:
        		ans.append('1')
        
        ans = int(str(int(''.join(ans))), 2)
        
        return ans
